modules/Bio/EnsEMBL/Compara/PipeConfig/ReindexMembers_conf.pm

```diff
@@ -342,6 +342,15 @@ sub core_pipeline_analyses {
             -flow_into      => {
                 '2->A' => 'exon_boundaries_prep',
                 'A->1' => 'pipeline_entry',
+                1      => "hc_deletion",
```

Comment:
Would it be possible to use single quotes here, to stay closer to the style of the rest of the pipe-config file?
```suggestion
                1      => 'hc_deletion',
```